src/cli/translate.ts

```
@@ -347,87 +479,119 @@ export const translateCommand = new Command('translate')
+              })
+
+              targetContent = await translate({
+                ...config.translate,
+                source,
+                sourceContent: normalizedSourceContent,
+                target,
+                additionalPrompts: sourceFrontmatter.i18n?.additionalPrompts,
+              })
+
+              const newFrontmatter = { ...sourceFrontmatter, sourceSHA }
+              delete newFrontmatter.i18n
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)